### ISSUE TITLE
exit with an error message when `oc` isn't running

### ISSUE THREAD

**Issue by ashetty1** [redhat-developer/ocdev#78]
When `oc cluster` isn't running, we should display a proper error message that the cluster isn't up and running;

```
# ocdev application create
Creating application: app
unable to create application: app: unable to create new project: command: [/usr/bin/oc new-project app] failed to run:
error: Missing or incomplete configuration info.  Please login or point to an existing, complete config file:

  1. Via the command-line flag --config
  2. Via the KUBECONFIG environment variable
  3. In your home directory as ~/.kube/config

To view or setup config directly use the 'config' command.
: exit status 1
```

**Comment by containscafeine** [comment]
needs to build on top of #99

**Comment by containscafeine** [comment]
@ashetty1 can you confirm this is fixed? I am getting the following output now -
```console
$ ocdev application create newapp
Creating application: newapp
unable to create new application: unable to get active application: unable to get current project info: please log in to the cluster
```

**Comment by containscafeine** [comment]
Should have been fixed in #99

**Comment by ashetty1** [comment]
Yes, fixed now :)
